include/render/gles2.h

```diff
@@ -56,11 +58,15 @@ struct wlr_gles2_renderer {
 			GLuint program;
 			GLint proj;
 			GLint color;
+			GLint pos_loc;
+			GLint tex_loc;
 		} quad;
 		struct {
 			GLuint program;
 			GLint proj;
 			GLint color;
+			GLint pos_loc;
+			GLint tex_loc;
```

Comment:
I don't think the ellipse shader uses texcoords

Reply:
The ellipse shader uses [`ellipse_fragment_src`](https://github.com/swaywm/wlroots/blob/master/render/gles2/renderer.c#L720), which uses the [`texcoord`](https://github.com/swaywm/wlroots/blob/master/render/gles2/shaders.c#L35) attribute

Reply:
Ah, indeed.